### DISCUSSION TITLE
Possible to redirect requests to different routes?

### DISCUSSION THREAD

**Discussion by jaxernst** [wevm/frog-archived#143] [General]
Say base route is being called, but the farcaster user has already 'authenticated' with my frame, and I want to redirect them to their dashboard. It would be nice if I can just forward that request to treat it as a different route.

I know I could conditionally call a different route handler function, but this seems to get a bit messy especially with getting the types right.

Anyone else found a good way to deal with this?

**Comment by ggomaeng** [comment]
You could probably create a [middleware](https://frog.fm/concepts/middleware) that checks the authenticated state and conditionally return another frame.
